iip-33.md

```diff
@@ -1,38 +1,116 @@
 ```
-IIP: 33
 Title: The DePIN DAO
-Author: Qevan Guo, Andrew Law
+Authors: Qevan Guo, Andrew Law
```

Comment:
add one line:
Forum discussion: https://community.iotex.io/t/iip-33-the-depin-dao/12323